core/backend/src/IModelJsFs.ts

```diff
@@ -143,7 +143,13 @@ export class IModelJsFs {
     const parentPath = path.dirname(dirPath);
     if (parentPath !== dirPath)
       IModelJsFs.recursiveMkDirSync(parentPath);
-    IModelJsFs.mkdirSync(dirPath);
+    try {
+      IModelJsFs.mkdirSync(dirPath);
+    } catch (err: any) {
+      // Ignore the error if the folder has been created since the existence check
+      if (err?.code !== "EEXIST")
```

Comment:
wait, typo? is the error code not `EXIST`?

Reply:
nvm, just read the docs, resolving

Reply:
Not a typo. See the error message on the associated [issue](https://github.com/iTwin/itwinjs-core/issues/7452) and the [Node docs](https://nodejs.org/api/errors.html#common-system-errors).